packages/api/src/utils/ethereum.ts

```diff
@@ -0,0 +1,53 @@
+const MIN_DATA_GASPRICE = 1;
+const DATA_GASPRICE_UPDATE_FRACTION = 3_338_477;
```

Comment:
```suggestion
const MIN_BLOB_GASPRICE = 1;
const BLOB_GASPRICE_UPDATE_FRACTION = 3_338_477;
```

Reply:
Following the recent name update: https://github.com/ethereum/EIPs/commit/58e82a0addec750bc3799b05f771cd8ab3c7fae1